pkg/storage/storage_test.go

```diff
@@ -84,19 +84,17 @@ var _ = Describe("In-memory storage", func() {
 
 		By("making sure that the storage contains all nodes received")
 		for _, node := range batch.Nodes {
-			ts, metrics := storage.GetNodeMetrics(node.Name)
-			Expect(ts).To(HaveLen(1))
-			Expect(metrics).To(HaveLen(1))
+			_, _, err := storage.GetNodeMetrics(node.Name)
+			Expect(err).NotTo(HaveOccurred())
 		}
 
 		By("making sure that the storage contains all pods received")
 		for _, pod := range batch.Pods {
-			ts, metrics := storage.GetContainerMetrics(apitypes.NamespacedName{
+			_, _, err := storage.GetContainerMetrics(apitypes.NamespacedName{
 				Name:      pod.Name,
 				Namespace: pod.Namespace,
 			})
-			Expect(ts).To(HaveLen(1))
```

Comment:
I think we still should test len of `ts` and `metrics`

Reply:
Done